gc/gc-base/src/main/java/org/projectnessie/versioned/persist/gc/BaseExpiredContentValues.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.persist.gc;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.projectnessie.model.Content;
+
+public final class BaseExpiredContentValues extends ExpiredContentValues {
```

Comment:
naming: a "base" class extends another class? This looks odd to me. Usually "base" classes are extended by other classes :thinking: